dev-start.sh

```
@@ -0,0 +1,160 @@
+fi
+
+# Check if npm is available
+if ! command -v npm >/dev/null 2>&1; then
+    echo "❌ Error: npm is not installed or not in PATH"
+    echo "💡 Please install Node.js and npm first"
+    exit 1
+fi
+
+# Start backend in background
+echo "🚀 Starting backend server..."
+if ! npm start &; then
```

Shellcheck (0.10.0)

[error] It's not 'foo &; bar', just 'foo & bar'.

(SC1045)